package.json

```diff
@@ -23,6 +24,6 @@
     "node": "*"
   },
   "scripts": {
-    "test": "node test/test.js"
+    "test": "node test/test.js || true"
```

Comment:
actually what that does it keep the annoying npm warnings at the end hidden.  But I removed it.